php/Blockonomics.php

```diff
@@ -502,20 +502,30 @@ public function calculate_order_params($order){
     // Get order info for unused or new orders
     public function calculate_new_order_params($order){
         $wc_order = new WC_Order($order['order_id']);
-            $order['expected_fiat'] = $wc_order->get_total();
-            $order['currency'] = get_woocommerce_currency();
-            if(get_woocommerce_currency() != 'BTC'){
-                $responseObj = $this->get_price($order['currency'], $order['crypto']);
-                if($responseObj->response_code != 200) {
-                    exit();
-                }
-                $price = $responseObj->price;
-                $price = $price * 100/(100+get_option('blockonomics_margin', 0));
-            }else{
-                $price = 1;
+        global $wpdb;
+        $orderss_id = $wc_order->get_id();
+        $results = array();
```

Comment:
Initiating results array is not required as get_results will return an empty array if no results